pkg/core/database/interfaces.go

```diff
@@ -94,6 +94,8 @@ type Transaction interface {
 
 	FetchOutputExists(destkey []byte) (bool, error)
 
+	FetchOutputUnlockHeight(destkey []byte) (uint64, error)
```

Comment:
Could we add a bit of description/comments for each exported interface method?

Reply:
Good point, will add it.